src/commands/protocols/common/epoch-check-command.js

```diff
@@ -329,16 +329,17 @@ class EpochCheckCommand extends Command {
             hashFunctionId: agreement.hashFunctionId,
             epoch: agreement.currentEpoch,
             agreementId: agreement.agreementId,
+            assertionId: agreement.assertionId,
             stateIndex: agreement.stateIndex,
             closestNode: closestNode.index,
             leftNeighborhoodEdge: neighbourhoodEdges?.leftEdge.index,
             rightNeighborhoodEdge: neighbourhoodEdges?.rightEdge.index,
         };
 
         await this.commandExecutor.add({
-            name: 'submitCommitCommand',
-            sequence: [],
-            retries: COMMAND_RETRIES.SUBMIT_COMMIT,
+            name: 'simpleAssetSyncCommand',
```

Comment:
if the idea is to store all service agreements data into mysql instead of just the ones the node stores in triple store, then it might be sensible to measure and log how long the getEligibleServiceAgreements queries take, just to ensure they don't get too slow as the table grows.